src/backend/distributed/master/master_stage_protocol.c

```diff
@@ -191,10 +191,14 @@ master_append_table_to_shard(PG_FUNCTION_ARGS)
 	text *sourceTableNameText = PG_GETARG_TEXT_P(1);
 	text *sourceNodeNameText = PG_GETARG_TEXT_P(2);
 	uint32 sourceNodePort = PG_GETARG_UINT32(3);
+
 	char *sourceTableName = text_to_cstring(sourceTableNameText);
 	char *sourceNodeName = text_to_cstring(sourceNodeNameText);
 
-	char *shardName = NULL;
+	Oid shardSchemaOid = 0;
```

Comment:
> On the other hand names like shardQuotedQualifiedName may be excessive and I can change them to something like shardQualifiedName.

I think that we can use shardQualifiedName to maintain consistency in the codebase.